massa-client/base_config/config.toml

```diff
@@ -6,8 +6,8 @@ timeout = 1000
 ip = "127.0.0.1"
 private_port = 33034
 public_port = 33035
-api_port = 33036
```

Comment:
Where is defined the port for the net API now ?

Reply:
We don't integrate the new API in `massa-client` yet. 

https://github.com/massalabs/massa/blob/3751e18022f7c29f7ce6b292e4d146921ecd563d/massa-node/base_config/config.toml#L15